src/bin/pgcopydb/defaults.h

```diff
@@ -16,6 +16,8 @@
 #define calloc(m, n) GC_malloc((m) * (n))
 #define free(p) GC_free(p)
 #define realloc(p, n) GC_realloc((p), (n))
+#define strdup(p) GC_strdup(p)
+#define strndup(p, n) GC_strndup(p, n)
```

Comment:
@dimitri I didn't see the strdup being overridden to use GC_ variant. Was there a specific reason?

Reply:
No reason other than it was missed, please can you make a separate PR for that?

Reply:
@dimitri FYI https://github.com/dimitri/pgcopydb/pull/691